api/config/model.yaml

```diff
@@ -0,0 +1,34 @@
+save_dir: ./ckpts/v22_bigmodel_mid_newdropout
+data_folder: null
+data_name: flickr8k_1_cap_per_img_1_min_word_freq
+checkpoint_file: ./ckpts/BEST_checkpoint_flickr8k_1_cap_per_img_1_min_word_freq.pth
+word_embedding_weight: null
+word_map_file: ./ckpts/WORDMAP_flickr8k_1_cap_per_img_1_min_word_freq.json
+attention_dim: 512
+emb_dim: 512
+decoder_dim: 1024
+style_dim: 128
+dropout: 0.5
```

Comment:
There are a lot of configurations here that are relevant for training but not for inference. I think they should be removed.